### PR TITLE
Add soundmemes.cr

### PR DESCRIPTION
**Pull request by vladfaust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -494,6 +494,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [icr](https://github.com/crystal-community/icr) - Interactive console for Crystal (like IRB for Ruby)
  * [nes](https://github.com/romeroadrian/nes.cr) - A NES emulator
  * [shards.rocks](https://shards.rocks/) - Service that manages dependencies inspired by [Gemnasium](https://gemnasium.com/) and [David](https://david-dm.org/)
+ * [soundmemes.cr](https://github.com/vladfaust/soundmemes.cr) - Telegram Bot built on top of tele.cr
```
Comment (veelenga) [inline README.md]: If it makes sense, let's link directly to https://t.me/soundmemesbot, instead to GitHub repo.

Comment (RX14) [reply README.md]: That URL doesn't work for me (redirects to `tg://resolve?domain=soundmemesbot` which is useless if you don't have telegram), please keep this as-is.

Comment (vladfaust) [reply README.md]: I want to share my source code, there is MIT License

Comment (veelenga) [reply README.md]: @RX14 strange, because I don't have telegram either and it does not redirect me. But, anyway, I am ok to keep it as is. Thanks

**Comment by veelenga** [comment]
@vladfaust repo is not available anymore. Should we remove it from the list ?

**Comment by vladfaust** [comment]
Yep, I decided to close the source. I don't mind removing it *but* it's still a Crystal project

**Pull request merged**